README.md

```diff
@@ -55,9 +55,109 @@ $ yarn audit
 ```shell
 $ yarn audit:fix
 ```
+### Setup AWS SSO Login
+```shell
+$ aws configure sso
+```
+- Choose session name: _{enter your name}_
+- Accept authorisation in browser
+- Choose the AWS account in the dropdown
+- CLI default client Region: eu-west-2
+- CLI default output format: json
+- aws sso login --profile _{profile name provided for account}
 
-## Additional Info
+## Architecture Diagram
+<img src="img.png" alt="img.png" width="500"/>
+
+## How To Deploy
+### 1. Setup Base CloudFormation Stacks
+To deploy the base common CloudFormation stacks required created by Dev Platform prior to deploying the solution use the Stack Orchestation tool provided in our [stack-orchestration](stack-orchestration) directory and deploy the [production_bootstrap.sh](stack-orchestration%2Fproduction_bootstrap.sh) script.
```

Comment:
when you say "deploy" a script.. do you mean to say deploy the stack using the script? If it is about running the script to deploy those stacks, would you need to set up credentials first?

Reply:
yep I mean **_run_** the script to deploy the stack - I added in the steps below so you'd cd into stack orchestration and then run `aws configure sso` in terminal to set up credentials, but then don't know if I should move the  **Setup AWS SSO Login** down here instead - what do you think?